src/linear_algebra/basic.lean

```diff
@@ -73,6 +73,17 @@ lemma smul_sum {α : Type u} {β : Type v} {R : Type w} {M : Type y}
   c • (v.sum h) = v.sum (λa b, c • h a b) :=
 finset.smul_sum
 
+@[simp]
+lemma smul_sum' {α : Type u} {R : Type v} {M : Type w} {M₂ : Type x}
```

Comment:
This name should probably be `sum_smul_index_linear_map'` or similar, since the LHS is the same as `sum_smul_index`

Reply:
Maybe just `sum_smul_index'`?

Reply:
I just added a commit renaming to `sum_smul_index_linear_map'` since `sum_smul_index'` is already taken.